src/lib/markets/tx.ts

```diff
@@ -298,49 +189,43 @@ export async function buildSubmitVaaTx({ chainId, marketAddress, marketOracles,
 
 export type CancelOrderDetails = { market: Address; nonce: bigint } | OpenOrder
 
-export type BuildPlaceOrderTxArgs = {
-  pythClient: EvmPriceServiceConnection
+type BuildTriggerOrderBaseArgs = {
   address: Address
-  marketOracles?: MarketOracles
   marketAddress: Address
-  marketSnapshots?: MarketSnapshots
-  orderType: OrderTypes
-  limitPrice?: bigint
-  stopLoss?: bigint
-  takeProfit?: bigint
   side: PositionSide
-  collateralDelta?: bigint
   delta: bigint
-  positionAbs: bigint
   selectedLimitComparison?: TriggerComparison
```

Comment:
I think this would only be used for limit orders, let's move it to the limit order tx args and rename to triggerComparison

We should also make it required